app/components/provider_interface/organisation_permissions_setup_check_component.html.erb

```diff
@@ -0,0 +1,12 @@
+<% grouped_relationships_by_name.each do |main_provider_name, relationship_list| %>
+  <% if multiple_providers_to_set_up? %>
+    <h2 class="govuk-heading-m"><%= main_provider_name %></h2>
+  <% end %>
+  <% relationship_list.each do |relationship| %>
+    <%= render ProviderInterface::OrganisationPermissionsReviewCardComponent.new(
+      provider_user: current_provider_user,
+      provider_relationship_permission: relationship,
+      change_path: edit_provider_interface_organisation_permissions_setup_path(relationship, checking_answers: true),
+    ) %>
+  <% end %>
```

Comment:
This is not accessible in the current state. The review card component also renders an h2. This clashes with the main provider h2 in the case of multiple providers. Confirmed with design that the review card component should be able to conditionally render an h3 instead in this case